### ISSUE TITLE
Do we have to change the DB schema / tables of the blockchain LMDB file for Seraphis?

### ISSUE THREAD

**Issue by rbrunner7** [seraphis-migration/wallet3#11]
Following my proposal of preferring single-subject issues, I allow myself to turn the following question no 5 from @dangerousfreedom1984 in his issue #10 into a separate issue:

> 5\. I think it is also important to work in parallel in implementing the necessary structures to store the data in an actual blockchain style so we can have a testnet as soon as possible. If we agree with that structure shown in figures 1 and 2, we have to basically modify the [file](https://github.com/monero-project/monero/blob/master/src/blockchain_db/lmdb/db_lmdb.cpp#L182) so the DB schema would account for that, right?

**Comment by rbrunner7** [comment]
As far as I could see going over the existing DB code it does not care about the actual structure of transactions and other things, they all seem to be stored simply as "blobs", as series of bytes. Seems to me that if we manage to serialize Seraphis transactions, we can very well store them into the same tables that the blockchain DB already has, e.g. `txs_pruned` or `txs_prunable`.

In my issue #7 I sketch a possible way how to define something like a "meta transaction class" that can hold legacy as well as Seraphis transactions that could be objects that we store in LMDB databases like the blockchain file.

But of course many detail questions remain. Although @UkoeHB says in #9 that we should continue with pruning and prune Seraphis transactions as well, maybe it's not clear the right way to do it is to continue to use two separate tables `txs_pruned` plus `txs_prunable` as mentioned above.

And maybe there will be completely new things to store in the blockchain in connection with Seraphis transactions, that will need new tables, or at least new ways to lookup Seraphis transactions that did not yet play a role until now.

My current take: If we want a permanent blockchain to back something like a proof-of-concept / prototype Seraphis wallet to all learn more and get acquainted with UkoeHB's library we just freely define a new schema, just for that purpose. After that, many such detail questions about the "real" blockchain DB hopefully will have cleared up, and we can attack that more complex problem.
